packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -19,13 +22,21 @@ export type BookmarkFeedLayoutProps = {
   onSearchButtonClick?: () => unknown;
 };
 
+const SharedBookmarksModal = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "accountDetailsModal" */ './modals/SharedBookmarksModal'
```

Comment:
The chunk name should be changed

```suggestion
      /* webpackChunkName: "sharedBookmarksModal" */ './modals/SharedBookmarksModal'
```